features/topology-map/org.opennms.features.topology.app/pom.xml

```diff
@@ -55,6 +55,11 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.opennms.features.vaadin-components</groupId>
+            <artifactId>graph</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
This dependency must also be added to the `features/topology-map/features/*` modules. Probably the `runtime-base`, otherwise the dependencies of the feature is not set correctly.